content/patterns/listbox/examples/css/listbox-actions.css

```diff
@@ -0,0 +1,81 @@
+[role="option"] span.actions {
+  display: none;
+}
+
+[role="option"]:hover span.actions,
+[role="listbox"]:focus [role="option"].focused span.actions {
```

Comment:
```suggestion
```
I wonder if we should remove this css to prevent this overlap from happening. This way only the options being hovered will show the action buttons and prevent the overlap if one is selected and the user moves the mouse
https://github.com/user-attachments/assets/67b85d47-452f-4a83-bda9-4a4ad3f426d4